python.d/memcached.chart.py

```diff
@@ -24,7 +24,7 @@
 
 CHARTS = {
     'cache': {
-        'options': [None, 'Cache Size', 'kilobytes', 'Cache', 'memcached.cache', 'stacked'],
+        'options': [None, 'Cache Size', 'megabytes', 'Cache', 'memcached.cache', 'stacked'],
         'lines': [
             ['bytes', 'used', 'absolute', 1, 1048576],
```

Comment:
Usually I take id from data returned by application (memcached) and assign custom name. In this case memcached returns used bytes as `bytes`, but for user it is better to see it as `used`. But I changed it in [last commit to master branch](https://github.com/firehol/netdata/commit/7442a84d723eb8eada6ba58359d6e808a4c1e4df).